pkg/netdevice/netResourcePool_test.go

```diff
@@ -113,10 +113,12 @@ var _ = Describe("NetResourcePool", func() {
 
 			fake1 := &mocks.PciNetDevice{}
 			fake1.On("GetPciAddr").Return("0000:01:00.1").
-				On("GetVdpaDevice").Return(nil)
+				On("GetVdpaDevice").Return(nil).
```

Comment:
can you also please add a test to validate we have the RDMA info in the deviceSpec?